core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -923,11 +923,20 @@ public String toString() {
   public static final PropertyKey MASTER_HEARTBEAT_INTERVAL_MS =
       new Builder(Name.MASTER_HEARTBEAT_INTERVAL_MS)
           .setAlias(new String[]{"alluxio.master.heartbeat.interval.ms"})
-          .setDefaultValue("1sec")
+          .setDefaultValue("1min")
           .setDescription("The interval between Alluxio masters' heartbeats.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_HEARTBEAT_TIMEOUT_MS =
+      new Builder(Name.MASTER_HEARTBEAT_TIMEOUT_MS)
+          .setAlias(new String[]{"alluxio.master.heartbeat.timeout.ms"})
+          .setDefaultValue("15min")
```

Comment:
Why is this so long? Since we just use the master heartbeat to monitor liveness, I think it should be fine to make this shorter, like 1 minute.

Reply:
Change to 1 min, thanks